tests/functions/test_expand_grid.py

```diff
@@ -0,0 +1,318 @@
+import pandas as pd
+import pytest
+import numpy as np
+from string import ascii_lowercase, ascii_uppercase
+from pandas._testing import assert_frame_equal
+
+from janitor.functions import expand_grid
+
+from janitor.utils import _check_instance
+
+
+def test_not_a_dict():
+    """Test that entry(list) is not a dictionary"""
+    data = [60, 70]
+    with pytest.raises(TypeError):
+        assert expand_grid(others=data)
+
+
+def test_not_a_dict_1():
+    """Test that entry (dataframe) is not a dictionary"""
+    data = pd.DataFrame([60, 70])
+    with pytest.raises(TypeError):
+        assert expand_grid(others=data)
```

Comment:
Assert not necessary.

```suggestion
        expand_grid(others=data)
```